arangod/Agency/Node.cpp

```diff
@@ -226,31 +174,35 @@ Node& Node::operator=(Node const& rhs) {
   // 1. remove any existing time to live entry
   // 2. clear children map
   // 3. move from rhs to buffer pointer
-  // Must not move rhs's _parent, _observers
-  removeTimeToLive();
+  // Must not move rhs's _parent
```

Comment:
Why are these big changes in the Node object needed, and why are they needed in this PR?